### ISSUE TITLE
[CHAOSS WEB]: Distinguishable - Contrast (Minimum) 1

### ISSUE THREAD

**Issue by yigakpoa** [chaoss/Accessibility#7]
### Please state the WCAG Guideline this issue falls under.

Perceivable

### More on Guidelines

1.4 Distinguishable - [1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/quickref/#qr-visual-audio-contrast-contrast) 

### Please state the WCAG Level this issue falls under.

AA

### Description

All text has a minimum contrast of 4.5:1

### Location

1. "Homepage- hero section 3/3 “CHAOSS software” button"
2.  Home screen- Get started button
3.  Homepage- About CHAOSS button
4. Homepage- listen to Chaosscast button
5. Homepage- August 10,2023-Blogspot- News
6. Homepage- hero section ⅔
7. Homepage- body- read more button
8. Homepage- subscribe to chaosstube button
9.  Cookie setting- Accept all button

### Actual Result

1. "Very low contrast between text and background colors(2.06:1)"
2.  Very low contrast between text and background colors(2:1)
3. Very low contrast between text and background colors(2:1)
4. Very low contrast between text and background colors(2:1)
5. Very low contrast between text and background colors(2.84:1)
6. Very low contrast between text and background colors(1.11:1)
7. Very low contrast between text and background colors(2:1)
8. Very low contrast between text and background colors(3.06:1)
9. Very low contrast between text and background colors(3.13:1)

### Your Evaluation

The "Actual Result" in each corresponding "Location" above should have a minimum contrast of 4.5:1. 

cc @Toriasdesign 

### Who is the best person to correct this?

Designer

**Comment by LamiWilliams** [comment]
I'd like to work on this

**Comment by LamiWilliams** [comment]
Hi @yigakpoa @peculiaruc Could you provide the link to the file where I'll make the changes? Thanks

**Comment by yigakpoa** [comment]
@ElizabethN @Toriasdesign

**Comment by ElizabethN** [comment]
Hi @LamiWilliams ! That would be awesome. These files are all managed in our Wordpress Theme so I can provide you access to that if it's something you're familiar with!

**Comment by LamiWilliams** [comment]
Thanks @ElizabethN 

Is this something that requires working with code? I can work with WordPress, but my knowledge is limited to Elementor

**Comment by ElizabethN** [comment]
@LamiWilliams Right - so this is for the CHAOSS Website itself, which is fully based on Wordpress. We don't have any Figma files for this site, as all the pages are based on a Wordpress theme that's already been installed. (As opposed to the Badging site which was created from scratch, so there *are* figma files for that one). I hope that makes sense! 

I think these changes will have to be done in the Wordpress interface or possibly through the CSS files in the Wordpress code, depending on what the changes are. We are limited by the Wordpress Admin interface.

**Comment by LamiWilliams** [comment]
> Got it. Thanks for clarifying!
> 
> I'll be unable to make the changes on the website, but I can determine the new values and share them here. The new assignee who can code can then implement them on web

@ElizabethN

**Comment by LamiWilliams** [comment]
HI @peculiaruc @yigakpoa 

Here are my recommendations:
1. Change button background to #D41A5D (5.12)
2. Change text color to #111111 (7.1)
3. Change text color to #11111 (7.1)
4. Change text color to #11111 (7.1)
5. …
6. Add a transparent overlay with a color of #FFFFFF at 12% opacity (4:9)
7. Change text color to #D41A5D
8. Change button background to #D41A5D (5.12)
9. Change background color to #27CCC0 and text to #111111 (7.1)

I can’t seem to locate the text mentioned in 5.

**Comment by peculiaruc** [comment]
> HI @peculiaruc @yigakpoa 
> 
> Here are my recommendations:
> 1. Change button background to #D41A5D (5.12)
> 2. Change text color to #111111 (7.1)
> 3. Change text color to #11111 (7.1)
> 4. Change text color to #11111 (7.1)
> 5. …
> 6. Add a transparent overlay with a color of #FFFFFF at 12% opacity (4:9)
> 7. Change text color to #D41A5D
> 8. Change button background to #D41A5D (5.12)
> 9. Change background color to #27CCC0 and text to #111111 (7.1)
> 
> I can’t seem to locate the text mentioned in 5. 
> 
> 

@yigakpoa Could you please give more direction or explanation that will help here.

**Comment by yigakpoa** [comment]
Thank @LamiWilliams for this. I am closing this issue right now since it has been completed.
